spec/service_objects/get_url_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'spec_helper'
+require 'rails_helper'
+
+describe GetUrl do
+  it "should get right url from RedisBase" do
+    getter = GetUrl.new('UcjrC3')
+    expect(getter.perform('localhost:3000')[0]).to eq('https://www.railscarma.com')
```

Comment:
If you tested method return you have to test the whole return value. Not just part of it. 
It means if your method returns array you should compare it with array.